### DISCUSSION TITLE
2 spaces instead of 4?

### DISCUSSION THREAD

**Discussion by ipatch** [fish-shell/fish-shell#10683] [Q&A]
i primarily use 2 spaces to edit all my code files. and just a little annoyance i find is when i'm editing a command via the command line and i insert a new line on say a `for` loop for a simple command ie. say i'm removing excess tmux sessions, the following statement (inside the for loop) is intended with 4 spaces, but i'd much prefer 2 spaces, but searching through the issues i didn't find any open or closed issues on the subject. below is a little copy paste of what i'm describing.

```
for session in 2 3
        tmux kill-session -t $session
    end
```

i'd much prefer to see something like,

```
for session in 2 3
  tmux kill-session -t $session
end
```

it'd be nice if there was a env var i set in my `config.fish` to control the indentation done on the command line. and maybe there is and i just didn't find it in the documentation.

**Comment by faho** [comment]
There is no configuration for this, just as there isn't for fish_indent. This is the fish style, and the downsides of having multiple styles outweigh any possible benefit from any specific style.

The issues were mostly asking for tabs (which are worse for interactive use because we try to align with the prompt and that's impossible if tab width is configurable). See e.g. #9633.
